GenderPayGap.WebUI/Helpers/ReportingYearsHelper.cs

```diff
@@ -29,5 +30,18 @@ public static string FormatYearAsReportingPeriod(int reportingPeriodStartYear)
             return formattedYear;
         }
 
+        public static DateTime GetDeadlineForAccountingDate(DateTime accountingDate)
+        {
+            const int deadlineExtensionFor2020InMonths = 6;

```

Comment:
I'd have expected this to go at the top of the file and maybe be in all caps (but maybe I've spent too much time staring at Java)

More helpfully, I definitely would've expected it to be commonised between here and the Return model class